tests/modules/og_test/og_test.module

```diff
@@ -39,3 +44,23 @@ function og_test_entity_insert(EntityInterface $entity) {
     $membership->save();
   }
 }
+
+/**
+ * Implements hook_og_user_access_entity_operation_alter().
+ */
+function og_test_og_user_access_entity_operation_alter(AccessResultInterface &$access_result, CacheableMetadata &$cacheable_metadata, $context): void {
```

Comment:
Seems like I missed this the first time, do we care here if we pass exactely the same pointer? I assume it's sufficient if the argument points to the same instance.
```suggestion
function og_test_og_user_access_entity_operation_alter(AccessResultInterface $access_result, CacheableMetadata $cacheable_metadata, $context): void {
```

Reply:
Resolving this, it is no longer applicable now that the hook has been dropped in favor of an event subscriber.